### PR TITLE
Include the reason compilation failed for Kotlin

### PR DESCRIPTION
**Pull request by geoand**
Reported by @ia3andy who found it while coding on code.quarkus.io

### CODE REVIEW

File: extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java [hunk 1]
```diff
@@ -71,8 +71,8 @@ public boolean hasErrors() {
         public void report(CompilerMessageSeverity severity, String s, CompilerMessageLocation location) {
             if (severity.isError()) {
                 if ((location != null) && (location.getLineContent() != null)) {
-                    errors.add(String.format("%s%n%s:%d:%d", location.getLineContent(), location.getPath(), location.getLine(),
-                            location.getColumn()));
+                    errors.add(String.format("%s%n%s:%d:%d%nReason:%s", location.getLineContent(), location.getPath(), location.getLine(),
```
Comment (gsmet) [inline extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java]: Minor remark but should we maybe have a space avec the colon?

Comment (geoand) [reply extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java]: `avec` -> `does not compute for an english speaker` :P

Kidding aside, yeah that makes sense

Comment (geoand) [reply extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java]: Fixed

Comment (gsmet) [reply extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java]: Oh my, looks like it's Friday already :)

**Review by ia3andy — APPROVED** [review]
LGTM

**Pull request merged**